src/config_trapdoor.c

```diff
@@ -490,14 +489,12 @@ TbBool parse_trapdoor_trap_blocks(char *buf, long len, const char *config_textna
 
 TbBool parse_trapdoor_door_blocks(char *buf, long len, const char *config_textname, unsigned short flags)
 {
-  struct ManfctrConfig *mconf;
   struct DoorConfigStats *doorst;
-  long pos;
-  int i,k,n;
-  int cmd_num;
+
```

Comment:
@Trass3r second example.
I played a few games with this pull, and did not run into problems.